### ISSUE TITLE
New Cert Extractions Discovered

### ISSUE THREAD

**Issue by piepieninja** [pbrah/wpa_supplicant-udmpro#10]
Some new Cert Extractions & Rooting seem to have been discovered for the BGW210-700 RG / Router, this may help people so that they don't have to buy a NGV* system

Article: https://www.dupuis.xyz/bgw210-700-root-and-certs/

A script that might help automate it: https://github.com/iwleonards/extract-mfg

I have yet to test this, but will be doing so soon and can report back

**Comment by pbrah** [comment]
Nice find!  Once you've tested please report back and if it worked out for you then I will add it to the main README.md.

**Comment by piepieninja** [comment]
So far I've been able to extract the certs on Ubuntu 18.04 with a combo of the script linked above and the linux binary from here: https://www.devicelocksmith.com/2018/12/eap-tls-credentials-decoder-for-nvg-and.html , so I've been doing this on Ubuntu and not windows. I had to replace the `.exe` with the appropriate linux binary

I was getting an error `sh: 1: mfg_dat_decode: not found` right after 

but I just commented out the script stuff so it was like: 

```
    print("Running mfg_dat_decode")
    os.chdir(destination)
    os.system("mfg_dat_decode > ../Output_from_mfg_dat_decode.txt")

    print("Cleaning Up")
    #os.remove("mfg_dat_decode")
    #for file in glob.glob("*.tar.gz"):
    #    shutil.move(file, "../")
```

Then I just manually ran `./mfg_dat_decode > ../test.txt` and was able to get the certs and wpa guy 

about to test docker ...

**Comment by impala454** [comment]
I can confirm this is the method I recently used as well and it worked from a Windows box.

**Comment by piepieninja** [comment]
Yep this works! Tried the cert extractions mentioned before on both Ubuntu 18.04 and MacOS. I was able to get it to run with some small flag changes that were mentioned by others in other issues here.
